src/include/mca/mca.h

```diff
@@ -321,6 +321,82 @@ void transpose(Matrix<T> &a);
 template <class T, class O>
 void transpose(const Matrix<T> &a, Matrix<O> &output);
 
+/* Calculate the exponentiation of a square matrix, and store the result in matrix a
```

Comment:
update this line with `Calculate the exponentiation of a square matrix in place, which means a will be changed`